--- src/open_mower/launch/include/_comms.launch	2023-05-13 19:18:36.162306262 +0000
+++ src/open_mower/launch/include/_comms.launch-changed	2023-05-13 20:50:32.539722391 +0000
@@ -5,7 +5,7 @@
 <launch>
     <!-- TODO: Add parameter for simulation -->
 
-    <node pkg="mower_comms" type="mower_comms" name="mower_comms" output="screen">
+    <node pkg="worx_comms" type="worx_comms" name="worx_comms">
         <rosparam unless="$(eval env('OM_MOWER')=='CUSTOM')" file="$(find open_mower)/params/hardware_specific/$(env OM_MOWER)/comms_general_params.yaml"/>
         <rosparam unless="$(eval env('OM_MOWER')=='CUSTOM')" file="$(find open_mower)/params/hardware_specific/$(env OM_MOWER)/comms_$(env OM_MOWER_ESC_TYPE)_params.yaml"/>
         <rosparam if="$(eval env('OM_MOWER')=='CUSTOM')" file="$(env HOME)/mower_params/comms_params.yaml"/>
