--- src/open_mower/launch/open_mower.launch	2023-05-13 19:18:36.162306262 +0000
+++ src/open_mower/launch/open_mower.launch-changed	2023-05-13 20:30:54.959786501 +0000
@@ -1,4 +1,6 @@
 <launch>
+    <include file="$(find open_mower)/launch/include/imu.launch"/>
+
     <include file="$(find open_mower)/launch/include/_comms.launch"/>
     <include file="$(find open_mower)/launch/include/_move_base.launch"/>
     <include file="$(find open_mower)/launch/include/_localization.launch"/>
